### PR TITLE
translation_da

### PR DESCRIPTION
**Pull request by o0bonde**
i hope i dont mess anything up this time..

### CODE REVIEW

File: translations/translation_da.xml [hunk 1]
```diff
@@ -544,12 +529,6 @@ Disse baner hedder altid det samme:
 venstre(2), venstre(1), center, højre(1), højre(2).
 2 køretøjer vil have venstre(1) og højre(1), 4 vil have venstre(2) og højre(2).
 Med 3 eller 5 køretøjer får du også centermuligheden.
-
-Once a multi tool course is loaded you will need to select the lane for the driver.
-Those lanes are always named the same:
-left(2), left(1), center, right(1), right(2).
-2 vehicles will have left(1) and right(1), 4 will have left(2) and right(2).
-With 3 or 5 vehicles you will also gain the center option.
```
Comment (Tensuko) [inline translations/translation_da.xml:552]: Same here

**Review by Tensuko — COMMENTED** [review]
You should always make sure, that you have the latest file.

**Comment by o0bonde** [comment]
ï downloaded this zi
Skærmbillede 2022-05-16 183144
p

**Comment by o0bonde** [comment]
i see

'the green is the danish version 
the red is the orginal
 
the red did  i forget to delete in last update

**Comment by o0bonde** [comment]
Skærmbillede 2022-05-16 183609

**Comment by Tensuko** [comment]
> the green is the danish version
> the red is the orginal

red is deleted, green is changed/added

**Comment by Tensuko** [comment]
Well I push this, but you will have to check if everything is there...

**Comment by o0bonde** [comment]
yes ill do that

**Pull request merged**